financial-services-accelerator/components/org.wso2.financial.services.accelerator.common/src/main/java/org/wso2/financial/services/accelerator/common/extension/model/ServiceExtensionTypeEnum.java

```diff
@@ -28,6 +28,7 @@ public enum ServiceExtensionTypeEnum {
     PRE_CONSENT_RETRIEVAL("pre_consent_retrieval"),
     PRE_CONSENT_REVOCATION("pre_consent_revocation"),
     PRE_CONSENT_AUTHORIZATION("pre_consent_authorization"),
+    PRE_CONSENT_PERSISTENCE("pre-consent-persistence"),
```

Comment:
these are should be with underscore now

Reply:
Fixed by https://github.com/wso2/financial-services-accelerator/pull/439/commits/afaa341b84a97c94f08ada7f2329bd59d671a0e2